schema/url/pass/plugin-map-markers-overlay.raml

```diff
@@ -18,45 +18,48 @@ types:
         required: false
       xAxisVariable: 
         type: VariableSpec
-      # TODO rename to geoAggregateVariable after b57
-      facetVariable: 
-        type: VariableSpec[]
-        maxItems: 2
-        required: false
+      geoAggregateVariable: 
+        type: VariableSpec
       valueSpec: 
         type: ValueSpec
       longitudeVariable?:
         type: VariableSpec
       latitudeVariable?:
         type: VariableSpec
+      binSpec:
+        type: BinSpec
       viewport?:
         type: GeolocationViewport
 
-  PieplotConfig:
+  MapMarkersOverlayConfig:
     type: PlotConfig
     properties:
-      rankedValues:
+      rankedValues?:
```

Comment:
why did this become optional?

Reply:
I figured if we're passing back binned continuous data maybe we didn't need it? It was added so the client would know what legend entries it needed iirc. Though I suppose it wouldn't hurt to have in either case if we just wanted to be consistent, or had some other use case.